src/main/java/org/osiam/addons/self_administration/SelfAdministration.java

```diff
@@ -7,30 +7,32 @@
 import org.osiam.addons.self_administration.one_time_token.ScavengerTask;
 import org.osiam.addons.self_administration.service.ConnectorBuilder;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.MessageSource;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.context.web.SpringBootServletInitializer;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
-import org.springframework.context.support.ResourceBundleMessageSource;
-import org.springframework.core.io.ClassPathResource;
-import org.springframework.core.io.Resource;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.mail.javamail.JavaMailSender;
 import org.springframework.mail.javamail.JavaMailSenderImpl;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
-import org.springframework.validation.Validator;
-import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
 import org.springframework.web.filter.CharacterEncodingFilter;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
 
 import com.google.common.base.Strings;
 
-@Configuration
+@SpringBootApplication
 @EnableWebMvc
-@ComponentScan
-public class SelfAdministration extends WebMvcConfigurerAdapter {
+@PropertySource("classpath:addon-self-administration.properties")
+public class SelfAdministration extends SpringBootServletInitializer {
+
+    public static void main(String[] args) throws NoFatWarDeploymentSupportedException {
+        throw new NoFatWarDeploymentSupportedException();
```

Comment:
So, you delete this line and we try to run it the way discussed in the comments (but don't support it officially)?